views/dataprivacy/text_weitergabe-von-daten.hbs

```diff
@@ -16,8 +16,7 @@
 <p>Für die Nutzung interaktiver Inhalte externer Anbieter (z. B. Schulbuchverlage) werden teilweise Pseudonyme eingesetzt. Diese dienen nur dazu, um Lernfortschritte festzuhalten (z.B. User 123456 hat Aufgabe X beendet). Die Inhalte-Anbieter erhalten hingegen keine unmittelbar personenbezogenen Daten (z.B. Namen, Kontaktdaten usw.) und können daher keine direkten Rückschlüsse auf die realen Nutzer:innen der {{theme.title}} ziehen. Die Daten über die Nutzung digitaler Angebote (bspw. Testfragen) können von den Inhalte-Anbietern anonymisiert ausgewertet und zur Verbesserung ihrer Inhalte und Produkte eingesetzt werden. Folgende Inhalte-Anbieter erhalten derzeit von dem HPI aufgrund von Kooperationsverträgen pseudonymisierte nutzungsbezogene Daten:</p>
 <ul>
 <li>Bettermarks GmbH, Skalitzer Str. 85, 10997 Berlin („Bettermarks“);</li>
-<li>neXenio GmbH (Nexboard), Charlottenstr. 59, 10117 Berlin („neXenio“).</li>
 </ul>
 
 <h5>Weitergabe wegen gesetzlicher Verpflichtung</h5>
-<p>Darüber hinaus gibt Deine / Ihre Schule und das HPI keine personenbezogenen Daten an Dritte weiter, es sei denn, die Weitergabe erfolgt in Erfüllung einer gesetzlichen Verpflichtung (z. B. im Rahmen von strafrechtlichen Ermittlungen).</p>
\ No newline at end of file
+<p>Darüber hinaus gibt Deine / Ihre Schule und das HPI keine personenbezogenen Daten an Dritte weiter, es sei denn, die Weitergabe erfolgt in Erfüllung einer gesetzlichen Verpflichtung (z. B. im Rahmen von strafrechtlichen Ermittlungen).</p>
```

Comment:
```suggestion
<p>Darüber hinaus gibt Deine / Ihre Schule und das SVS keine personenbezogenen Daten an Dritte weiter, es sei denn, die Weitergabe erfolgt in Erfüllung einer gesetzlichen Verpflichtung (z. B. im Rahmen von strafrechtlichen Ermittlungen).</p>
```

Reply:
Soweit ich es sehe, ist alles unter dataprivacy veraltet und nicht mehr in Benutzung. Ich frag dazu mal an anderer Stelle nach. Ich würd da jetzt aber nicht noch mehr anpassen.